app/controllers/histories_controller.rb

```diff
@@ -1,5 +1,5 @@
 class HistoriesController < ApplicationController
-  before_action :load_bills, only: %i(index show)
+  before_action :logged_in_user, :load_bills, only: %i(index show)
```

Comment:
```suggestion
  before_action :logged_in_user
  before_action :load_bills, only: %i(index show)
```
ri hợp lý hơn
vào page này bắt buộc phải login =)) ko care đến action luôn

Reply:
dạ vag ạ